scripts/update-cli-help.js

```diff
@@ -0,0 +1,30 @@
+// Help doc on main
+const path = require("path");
+const fs = require("fs");
+
+const DOC_URL =
+  "https://raw.githubusercontent.com/enviodev/hyperindex/main/codegenerator/cli/CommandLineHelp.md";
```

Comment:
Should we maybe install `envio` package instead and call the binary directly?

Reply:
I tried but long story it breaks using yarn package manager (which is something we need to fix!)